components/org.wso2.openbanking.cds.identity/src/main/java/org/wso2/openbanking/cds/identity/metadata/periodical/updater/utils/Utils.java

```diff
@@ -112,4 +117,39 @@ private static JSONObject getResponseJson(HttpResponse response) throws IOExcept
         }
 
     }
+
+    /**
+     * Initializes the super tenant domain and tenant ID in the Carbon context if not already set.
+     */
+    public static void initializeTenantContextIfAbsent() throws OpenBankingException {
```

Comment:
Use MultitenantConstants as we need to set ST domain and id. You could get rid of this method.

Reply:
fixed